wagtail/admin/ui/side_panels.py

```diff
@@ -139,6 +139,7 @@ class BasePreviewSidePanel(BaseSidePanel):
     order = 400
     toggle_aria_label = gettext_lazy("Toggle preview")
     toggle_icon_name = "mobile-alt"
+    keyboard_shortcut = "mod+p"
```

Comment:
I personally think we should not override the print command shortcut but this is current state. 

Potential future improvement could be `]` or `mod+]` - Trello uses this to show the 'right' sidebar.

Reply:
Made the call to go with `alt+p` here as overriding the print function did not make sense.